ui/src/components/DeployCoreInstance.tsx

```diff
@@ -25,13 +25,35 @@ export default function DeployCoreInstance() {
         const args = [
             "apply",
             "--context", "docker-desktop",
-            "-f", "https://raw.githubusercontent.com/calyptia/vivo/master/vivo.yaml"
+            "-f", "https://raw.githubusercontent.com/calyptia/vivo/master/vivo-deployment.yaml"
```

Comment:
Vivo has changed quite a bit to support K8S so hopefully should be simpler now.
I have a deployment in on-prem for it: https://github.com/calyptia/chart-cloud-standalone/blob/main/chart/templates/vivo/deployment-vivo.yaml